name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/StatementOfAssetsViewer.java

```diff
@@ -746,6 +747,27 @@ private void addDividendColumns(List<ReportingPeriod> options)
         column.setSorter(ColumnViewerSorter.create(new ElementComparator(labelProvider)));
         column.setVisible(false);
         support.addColumn(column);
+
+        // dividend yield on costs auf der hauptübersicht/vermögensaufstellung
+        column = new Column("yocdiv", Messages.ColumnDividendYieldOnCost, SWT.RIGHT, 80); //$NON-NLS-1$
+        column.setOptions(new ReportingPeriodColumnOptions(Messages.ColumnDividendYieldOnCost + " {0}", // $NON-NLS-1$ //$NON-NLS-1$
+                        List.of(new SinceX(LocalDate.now().minusDays(365)))));
+        column.setGroupLabel(Messages.GroupLabelDividends);
+        column.setDescription(Messages.ColumnDividendYieldOnCost_Description);
+        column.setSorter(ColumnViewerSorter.create(new ElementComparator(labelProvider)));
```

Comment:
This uses the labelProvider of the previous column to sort this column

Reply:
Oh thx. Now I fixed it.